.githubworkflows/.githubworkflowslinters.yml.txt

```diff
@@ -0,0 +1,56 @@
+name: Linters
```

Comment:
- [ ] Kindly make sure to add the correct files for the linter set up, you have added the wrong file. Also, make sure all the checks are passing.

Reply:
Okay, I will check it. Thank you.